frontend/src/_services/app.service.js

```diff
@@ -76,6 +77,16 @@ function changeIcon(icon, appId) {
   return fetch(`${config.apiUrl}/apps/${appId}/icons`, requestOptions).then(handleResponse);
 }
 
+function renameApp(name, appId) {
```

Comment:
I think we already have a function for updating the app name. 
`appService.saveApp()`. The same function we are using inside the `EditAppName.jsx` component